sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowExecBase.scala

```diff
@@ -136,8 +136,8 @@ trait WindowExecBase extends UnaryExecNode {
           val frame = spec.frameSpecification.asInstanceOf[SpecifiedWindowFrame]
           function match {
             case AggregateExpression(f, _, _, _, _) => collect("AGGREGATE", frame, e, f)
-            case f: FrameLessOffsetWindowFunction =>
-              collect("FRAME_LESS_OFFSET", f.fakeFrame, e, f)
+            case f: FrameLessOffsetWindowFunction => collect(
+              s"FRAME_LESS_OFFSET_${f.ignoreNulls}_${f.input.prettyName}", f.fakeFrame, e, f)
```

Comment:
`prettyName` can be very long. Can we use the index of the input?

Reply:
Can't distinguish input completely according to index. Different window function may use the same input.

Reply:
maybe we should do a refactor here to make the key more flexiable. I don't think `Expression.prettyName` can be used as a key reliably.

Reply:
OK